data/styles/outdoors/include/Icons.mapcss

```diff
@@ -266,6 +266,10 @@ area|z16-[amenity=shelter],
 node|z16-[amenity=shelter][shelter_type=lean_to],
 node|z16-[amenity=shelter][shelter_type=basic_hut],
 {font-size: 12; icon-min-distance: 0;}
+node|z13-[amenity=shelter][shelter_type=public_transport],
+{icon-image: zero-icon.svg;}
```

Comment:
please avoid using a zero-icon hack (e.g. its user-selectable still)
please use `icon-image: none;` instead

Reply:
and `node|z13[amenity=...` is enough as we need to reset it for z13 only

Reply:
Thanks. I didn't know that you could set the icon-image to none

Reply:
yeap its a relatively new improvement :)
need to disable the caption also..